addons/mail/models/mail_canned_response.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+
+
+class MailCannedResponse(models.Model):
+    """
+    Canned Response: content that will automatically replace the shortcut of your choosing. This content can still be adapted before sending your message.
+    """
+
+    _name = "mail.canned_response"
+    _description = "Canned Response"
+    source = fields.Char(
+        "Shortcut",
+        required=True,
+        index="trigram",
+        help="Canned response that will automatically be substituted with longer content in your messages."
+        " Type ':' followed by the name of your shortcut (e.g. :hello) to use in your messages.",
```

Comment:
We have some space on our screens, no need to force one attribute / line
```suggestion
    source = fields.Char(
        "Shortcut", required=True, index="trigram",
        help="Canned response that will automatically be substituted with longer content in your messages."
        " Type ':' followed by the name of your shortcut (e.g. :hello) to use in your messages.",
```